### PR TITLE
Create components canvas inputs

### PR DESCRIPTION
**Pull request by chris-kuehn**

### CODE REVIEW

File: components/CanvasOne.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function CanvasOne({
+  canvasOneParameter,
+  // handleColorChange,
+  // handleRotationChange,
+  // handleOutlineChange,
+  // handleVariation,
```
Comment (mbosselmann) [inline components/CanvasOne.js:9]: Outcommented code should be removed :)

File: components/CanvasOne.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      context.lineWidth = canvasOneParameter[3].lineWidthes[i];
+      context.strokeRect(coordinateOnXAxis, coordinateOnYAxis, width, height);
+    }
+
+    // Reset the transformation matrix to the identity matrix
+    context.setTransform(1, 0, 0, 1, 0, 0);
+    // End drawing statements
+  }, [canvasOneParameter]);
+  return (
+    <>
```
Comment (mbosselmann) [inline components/CanvasOne.js:45]: Fragments are not needed. 👀

File: components/CanvasOne.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function CanvasOne({
```
Comment (mbosselmann) [inline components/CanvasOne.js:4]: Why is the name of the component `CanvasOne`?  Do you have more than one canvas component? 🤔 Usually we prefer to not use numbers for the naming of your components. If a number indicates a type of order of components it reduces the possible reusability of your component. :)

File: components/CanvasOne.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  canvasOneParameter,
```
Comment (mbosselmann) [inline components/CanvasOne.js:5]: Same here for this parameter name that is related to your component name.

File: components/InputsOne.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function InputsOne({
```
Comment (mbosselmann) [inline components/InputsOne.js:4]: Same here for `InputsOne` as mentioned above. As this is the configuration part for your art generator maybe a more suitable as `InputsOne` can be found. :)

File: components/InputsOne.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        id="outline"
+      />
+      <StyledVariatonButton onClick={() => handleVariation()}>
+        Variation
+      </StyledVariatonButton>
+    </StyledInputWrapper>
+  );
+}
+
+const StyledInputWrapper = styled.div`
```
Comment (mbosselmann) [inline components/InputsOne.js:47]: Later when it is possible to add the art work to the list of art works (if I remember your plan correctly), this should be a `form`. :)

File: components/InputsOne.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+      <label htmlFor="outline">Amount of outlines </label>
+      <StyledInputRange
+        type="range"
+        min="0"
+        max="8"
+        value={canvasOneParameter[2].outline}
+        onChange={(event) => handleOutlineChange(event.target.value)}
+        id="outline"
+      />
+      <StyledVariatonButton onClick={() => handleVariation()}>
```
Comment (mbosselmann) [inline components/InputsOne.js:40]: You should add `type="button"` as a button is by default a submit button.

File: pages/index.js [hunk 8]
```diff
@@ -1,17 +1,96 @@
-import Canvas from "@/components/Canvas";
+import React, { useState, useRef } from "react";
```
Comment (mbosselmann) [inline pages/index.js:1]: You can remove "React" from the import. :)

**Review by mbosselmann — COMMENTED** [review]
Hi Christian,
great job! 💪 Left some remarks for you to consider. :)

File: components/CanvasOne.js (hunk 1 above)
Comment (chris-kuehn) [reply components/CanvasOne.js:9]: ups ... sorry! :)

File: components/CanvasOne.js (hunk 2 above)
Comment (chris-kuehn) [reply components/CanvasOne.js:45]: Deleted! :)

File: components/InputsOne.js (hunk 7 above)
Comment (chris-kuehn) [reply components/InputsOne.js:40]: Added! :)

File: components/InputsOne.js (hunk 6 above)
Comment (chris-kuehn) [reply components/InputsOne.js:47]: will do that in the next user story :) Liked we talked about it!

File: components/InputsOne.js (hunk 5 above)
Comment (chris-kuehn) [reply components/InputsOne.js:4]: Deleted "ONE"! 😩

File: components/CanvasOne.js (hunk 4 above)
Comment (chris-kuehn) [reply components/CanvasOne.js:5]: Deleted "ONE"! 😩

**Pull request merged**